src/Animals/Ostrich.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Animals;
+
+class Ostrich extends Bird
+{
+    private $speed;
```

Comment:
But in my Ostrich class I have changed the logic of the setSpeed method. So I can't use speed trait. Am I wrong?